reference/providers/kubernetes-v2/index.md

```diff
@@ -128,6 +128,14 @@ command](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands
   ConfigMap or Secret to be deployed without appending a new version when the
   contents change, for example.
 
+* `strategy.spinnaker.io/use-source-capacity`
+
+  When set to `'true'` or `'false'`, this overrides the resources replicas size
```

Comment:
s/resources/resource's/
and
s/replicas/replica/  
?

Or should it be "...overrides the resource replica's size..."?

Reply:
I think it should be
"this overrides the resource's replica count"